chart/templates/bioscfg-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bioscfg-config
+  namespace: default
+data:
+  config.yaml: |
+    log_level: debug
+    concurrency: 5
+    nats:
+      app_name: bioscfg
+      # TODO: remove deprecated stream_urn_ns field
+      stream_urn_ns: hollow-controllers
```

Comment:
copypasta? you shouldn't need `stream_urn_ns`

Reply:
Awesome, thanks for the info!